### PR TITLE
[翻译] 08-apollo-server 第9部分

### LINKED ISSUE #10: [翻译] 08-apollo-server
## 翻译范围

| 章节  | 文件 | 文件行数 |
| ------------- | ------------- |-------------|
| [08-apollo-server](https://github.com/the-road-to-graphql/the-road-to-graphql-chinese/tree/master/manuscript/08-apollo-server) | [index-cn.md](https://github.com/the-road-to-graphql/the-road-to-graphql-chinese/blob/master/manuscript/08-apollo-server/index-cn.md) | 5338 |

## 翻译人员 （需要10+人）

负责人：@scglwsj

第一部分 @scglwsj  
第二部分 @clattanoia
第三部分 @ListenSong1013
第四部分 @WarnerHooh
第五部分 @sheltonsuen
第六部分 @HazelZ
第七部分 @Mandy-Tang
第八部分 @Heyff12
第九部分 @SimonTal
第十部分 @zhangyouxin

## 校对人员

第一部分 @liujiefengfeng @SimonTal
第二部分 @scglwsj
第三部分 @sheltonsuen
第四部分 @ListenSong1013
第五部分 @xbini
第六部分 @qiuyunfen
第七部分 @Heyff12
第八部分 @Mandy-Tang 
第九部分 @HazelZ @zhangyouxin 
第十部分 @futantan

### PR DESCRIPTION
**Pull request by SimonTal**
//resolve #10 / 9

### CODE REVIEW

File: manuscript/08-apollo-server/index-cn.md [hunk 1]
```diff
@@ -4309,9 +4319,13 @@ export default {
...
+
+subscribe 函数签名和其他解析函数相同。上下文提供的模型对象可以在这里访问，但是在本例中用不到。
 
-The subscription as resolver provides a counterpart for the subscription in the message schema. However, since it uses a publisher-subscriber mechanism (PubSub) for events, you have only implemented the subscribing, not the publishing. It is possible for a GraphQL client to listen for changes, but there are no changes published yet. The best place for publishing a newly created message is in the same file as the created message:
+> The subscription as resolver provides a counterpart for the subscription in the  . However, since it uses a publisher-subscriber mechanism (PubSub) for events, you have only implemented the subscribing, not the publishing. It is possible for a GraphQL client to listen for changes, but there are no changes published yet. The best place for publishing a newly created message is in the same file as the created message:
+
+Subscription 里的解析器处理了 message schema 中所对应的订阅。上面的代码完成了消息的订阅。然而，在发布订阅模型中，你还需要实现发布。在订阅解析器的同一个文件中放置发布函数是最好的。
```
Comment (HazelZ) [inline manuscript/08-apollo-server/index-cn.md:4329]: It is possible for a GraphQL client to listen for changes, but there are no changes published yet.
这一句是认为无关被直接省略了吗？

Comment (SimonTal) [reply manuscript/08-apollo-server/index-cn.md:4329]: > you have only implemented the subscribing, not the publishing. It is possible for a GraphQL client to listen for changes, but there are no changes published yet. 

个人觉得这两句所表达的意思大致相同，所以我觉没有再翻译第二句了。你有什么好的建议吗？

**Comment by SimonTal** [comment]
@HazelZ 如果没有其他疑问，我就准备 merge 进 master 了。

**Comment by scglwsj** [comment]
好的话 @我 一下，我就合并了

**Pull request merged**